modules/@apostrophecms/page-type/index.js

```diff
@@ -173,7 +173,7 @@ module.exports = {
           });
           if (published && (doc.level > 0)) {
             const { lastTargetId, lastPosition } = await self.apos.page
-              .inferLastTargetIdAndPosition(doc);
+              .inferLastTargetIdAndPosition(doc, { publishedTargetsOnly: true });
```

Comment:
Page tree editing occurs in draft, but we replay the movement afterwards in the published locale. When doing so we need to make sure we only look at published targets.